src/Compiler/RefC/RefC.idr

```diff
@@ -131,23 +98,22 @@ where
 
 
 cConstant : Constant -> String
-cConstant (I x) = "(Value*)makeInt32("++ show x ++")" -- (constant #:type 'i32 #:val " ++ show x ++ ")"
-cConstant (BI x) = "(Value*)makeInt64("++ show x ++")" --"(constant #:type 'i64 #:val " ++ show x ++ ")"
-cConstant (Db x) = "(Value*)makeDouble("++ show x ++")"--"(constant #:type 'double #:val " ++ show x ++ ")"
-cConstant (Ch x) = "(Value*)makeChar("++ escapeChar x ++")" --"(constant #:type 'char #:val " ++ escapeChar x ++  ")"
+cConstant (I x) = "(Value*)makeInt32("++ show x ++")"
+cConstant (BI x) = "(Value*)makeInt64("++ show x ++")"
+cConstant (Db x) = "(Value*)makeDouble("++ show x ++")"
+cConstant (Ch x) = "(Value*)makeChar("++ escapeChar x ++")"
 cConstant (Str x) = "(Value*)makeString("++ cStringQuoted x ++")"
-  -- = "(constant #:type 'string #:val " ++ cStringQuoted x ++ ")"
 cConstant WorldVal = "(Value*)makeWorld()"
 cConstant IntType = "i32"
 cConstant IntegerType = "i64"
 cConstant StringType = "string"
 cConstant CharType = "char"
 cConstant DoubleType = "double"
 cConstant WorldType = "f32"
-cConstant (B8 x)   = "(Value*)makeInt8("++ show x ++")" --"(constant #:type 'i64 #:val " ++ show x ++ ")"
-cConstant (B16 x)  = "(Value*)makeInt16("++ show x ++")" --"(constant #:type 'i64 #:val " ++ show x ++ ")"
-cConstant (B32 x)  = "(Value*)makeInt32("++ show x ++")" --"(constant #:type 'i64 #:val " ++ show x ++ ")"
-cConstant (B64 x)  = "(Value*)makeInt64("++ show x ++")" --"(constant #:type 'i64 #:val " ++ show x ++ ")"
+cConstant (B8 x)   = "(Value*)makeInt8("++ show x ++")"
```

Comment:
Are these unsigned int types? In this case we should probably call the function `makeUInt8`.
Same question for the rest.

Reply:
They are stored internally as signed. I was considering changing them to unsigned, and renaming to `makeBits8` to more closely match the Idris types they represent, but this PR is already quite big.

I'm planning another PR for bigint support - I could include it in that.